### PR TITLE
CLDR-15276 Update hi_Latn inheritance

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-15276

Update hi_Latn inheritance to en_IN

[ ] This PR completes the ticket.

ALLOW_MULTI_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
may need to squash it anyway to fix the commit message

**Comment by srl295** [comment]
```xml
			<language type="vec">Vietnamese</language>
```

Whoops ^

**Comment by AEApple** [comment]
> ```
> 			<language type="vec">Vietnamese</language>
> ```
> 
> Whoops ^

Thanks! I will fix that.

**Comment by srl295** [comment]
> > ```
> > 			<language type="vec">Vietnamese</language>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Whoops ^
> 
> 
> Thanks! I will fix that.

Already working on it

There are a couple like that.

File: seed/main/hi_Latn.xml [hunk 1]
```diff
@@ -5279,11 +5278,6 @@ annotations.
 				<unitPattern count="other">{0} gal</unitPattern>
 				<perUnitPattern>{0}/gal</perUnitPattern>
 			</unit>
-			<unit type="volume-quart">
-				<displayName>qts</displayName>
-				<unitPattern count="one">{0} qt</unitPattern>
-				<unitPattern count="other">{0} qt</unitPattern>
-			</unit>
```
Comment (srl295) [inline seed/main/hi_Latn.xml]: this conflicts with en_001 (and en_IN) where these are *US* quarts and imperial are just *quarts*. So, deleted.

File: seed/main/hi_Latn.xml [hunk 2]
```diff
@@ -512,7 +512,6 @@ annotations.
 			<language type="uz">Uzbek</language>
 			<language type="vai">Vai</language>
 			<language type="ve">Venda</language>
-			<language type="vec">Vietnamese</language>
```
Comment (srl295) [inline seed/main/hi_Latn.xml]: oops

Comment (AEApple) [reply seed/main/hi_Latn.xml]: I would expect that these would cause collisions when inheritance is  included. Is that not happening?

Comment (srl295) [reply seed/main/hi_Latn.xml]: i deleted `vec` so that it's no longer an issue

Comment (AEApple) [reply seed/main/hi_Latn.xml]: Okay, but it would cause collisions for any other instances of the wrong line in the xml, right?

Comment (srl295) [reply seed/main/hi_Latn.xml]: @AEApple  this line was deleted

**Review by srl295 — COMMENTED** [review]
Someone else should review my code

**Comment by AEApple** [comment]
> Someone else should review my code

Makes sense!

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**